SketchUp/Sketchup/FrameChangeObserver.rb

```diff
@@ -61,33 +71,37 @@ class Sketchup::FrameChangeObserver
   #
   # @example
   #   class MyFrameChangeObserver
-  #     def frameChange(from_page, to_page, percent_done)
-  #       puts "From page #{from_page.name} to #{to_page.name} (#{percent_done * 100}%)"
+  #     def frameChange(from_scene, to_scene, percent_done)
+  #       if percent_done == 0.0
+  #         puts "Animating to scene '#{to_scene.name}':"
+  #       else
+  #         puts format("% 7.2f %",percent_done*100)
+  #       end
   #     end
   #   end
-  #   id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
+  #   @id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
   #
   # @note The +from_scene+ argument into this callback does not appear to
   #   be populated on the PC. You can store a variable that keeps track of
   #   the +to_scene+ and then use that on a subsequent Scene selection to
   #   determine the last Page that the user was on.
   #
-  # @param [Float] percent_done
-  #   The percentage of transition between
-  #   the two scene pages.
+  # @param from_scene [Sketchup::Page, nil]
```

Comment:
`#to_raw` looks like it would be used for outputting text (the stubs?)

> When YARD reformats a docstring it use type before name.

It **doesn't** when outputting the html...
Well, the name first way works fine for outputting the html pages direct from Ruby stubs. I know that YARD must be flexible enough not to do what it says in the default `#to_raw`.  But you'd have to subclass the DocstringParser and override the method.

> And we have a bunch of code already that follow this format.

But it WOULDN'T follow the old format if you did subclass and override, then one cycle through the stubs and back and out to stubs again, and it'd be name first from then on. No manually having to change then right ?

Which really could be done at any time in the future, right ?